git/README.md

```diff
@@ -0,0 +1,35 @@
+# Git Source Control Management
+
+### Table of contents
+1.	[Branches](#branches)
+
+#### Branches
+A project should at least have the following branches
+- [master](#master)
+- [staging](#staging)
+- [develop](#develop)
+- [feature](#feature)
```

Comment:
`feature` shouldn't be a branch itself, its kind of a type of branch you'd have if you're following topical branches model.